### PR TITLE
add install instruction

### PR DESCRIPTION
**Pull request by zhsj**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,3 +5,41 @@
...
 repository to make packaging in various Linux distributions easier.
+
+# Install instruction
+
+You need to have `dkms` and linux-headers on your system. You can install them by
+`sudo apt install dkms` or `sudo yum install dkms` (`dkms` is available in epel repo
+for CentOS).
+
+Package name for linux-headers varies on different distributions, e.g.
+linux-headers-generic(Ubuntu), linux-headers-amd64(Debian),
```
Comment (morphis) [inline README.md]: Lets write this as:  `linux-headers-generic` (Ubuntu), ...

File: README.md [hunk 2]
```diff
@@ -5,3 +5,41 @@
...
+  ```
+
+* Finally use `dkms` to build and install:
+
+  ```
+  sudo dkms install anbox-ashmem/1
+  sudo dkms install anbox-binder/1
+  ```
+
+Try to load the modules to verify:
```
Comment (morphis) [inline README.md]: A quick test to check if everything went fine is to do a 

`$ ls -alh /dev/binder /dev/ashmem`

so users can compare permissions etc.

**Comment by zhsj** [comment]
@morphis ping?

**Comment by morphis** [comment]
Thanks!

**Pull request merged**